.github/workflows/test-database.yml

```diff
@@ -70,6 +70,11 @@ jobs:
           docker compose --profile mssql --profile postgres --profile mysql up -d
         shell: pwsh
 
+      - name: Add Maria DB
+        # Use a different port than MySQL
+        run: |
+          docker run --detach --name mariadb --env MARIADB_ROOT_PASSWORD=mariadb-password -p 4306:3306 mariadb:latest
```

Comment:
❓ Should we use 10 like our docs have?

Reply:
Yes, probably, but the architecture of these tests means we could test multiple versions if we wanted to.